data-sources/lighttpd/config.yml

```diff
@@ -0,0 +1,22 @@
+id: lighttpd
+displayName: Lighttpd
+description: |
+  ## Comprehensive monitoring for your lighttpd server
```

Comment:
Data source descriptions don't support markdown and are expected to be a short description. Please come up with a short/concise description for this data source